src/v/wasm/wasmtime.cc

```diff
@@ -54,6 +54,7 @@
 namespace wasm::wasmtime {
 
 namespace {
+
```

Comment:
For my own sanity, this commit is just moving definitions and declarations around yeah?

Reply:
GitHub's review tool is sort of a 🗑️ 🔥 so I'm assuming you're referring to f92b37cbb21240374796a62c4645567605278a18 here, if so then yes. The commit just moves the definition up so that in later commits the engine can use a fully defined runtime. It's enough noise that it's nice to pull into a separate commit.

Reply:
>GitHub's review tool is sort of a 🗑️ 🔥 so I'm assuming you're referring to https://github.com/redpanda-data/redpanda/commit/f92b37cbb21240374796a62c4645567605278a18 here

Ugh, yes correct. I wasn't suggesting that it _shouldn't_ be a separate commit, just flagging that I didn't go through it line by line.